solr/solrj/src/test/org/apache/solr/client/solrj/SolrExampleTests.java

```diff
@@ -710,10 +714,18 @@ public void testContentStreamRequest() throws Exception {
     Assert.assertEquals(0, rsp.getResults().getNumFound());
 
     ContentStreamUpdateRequest up = new ContentStreamUpdateRequest("/update");
-    up.addFile(getFile("solrj/books.csv"), "application/csv");
+    // Create a copy of the file, which can be deleted after uploading.
+    // If the stream isn't closed, the file deletion will fail on Windows,
+    // though it will succeed on linux regardless
+    final File file = new File("temp/books_copy.csv");
```

Comment:
I see you're creating a file relative to the current working directory.  That's a bad practice for tests generally.  Call `createTempFile(...)` instead (declared in the inheritance chain).

Reply:
Thanks, for some reason I thought that createTempFile() wouldn't work for what I was doing (too late at night). I've changed that